app/components/TracksCard/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ * TracksCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Image } from 'antd';
+import If from '@components/If';
+import { isEmpty } from 'lodash';
+
+const CustomCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    width: 240px;
+    height: 600px;
+  }
+`;
+
+const CustomText = styled.p`
+  margin: 10px;
+  font-size: 1rem;
+`;
+
+export function RepoCard({ collectionName, trackName, artworkUrl100: url }) {
```

Comment:
is it RepoCard or trackCard ? Instead of artWorkUrl100. change the prop name to `image`